cp3-networkpolicy/egress/license-service-reporter/ibm-license-service-reporter-operator-egress.yaml

```diff
@@ -3,15 +3,13 @@ kind: NetworkPolicy
 metadata:
   name: egress-ibm-license-service-reporter-operator
   namespace: "lsrNamespace"
-  labels:
```

Comment:
pls restore the labels.
please DO NOT rename filename.

Reply:
done